doc/rst/language/spec/ranges.rst

```diff
@@ -465,14 +465,22 @@ undefined behavior.
    checked at runtime and the program will halt if the range iteration is
    invalid.
 
-.. _Iterating_over_Unbounded_Ranges_in_Zippered_Iterations:
+.. _Iterating_over_Unbounded_Ranges:
 
-Iterating over Unbounded Ranges in Zippered Iterations
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Iterating over Unbounded Ranges
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-When a range with the first index but without the last index is used in
-a zippered iteration ( :ref:`Zipper_Iteration`), it generates as
-many indices as needed to match the other iterator(s).
+When an unbounded range of integer values is used to drive a loop,
+either by being the only iterand, or by serving as the leader iterand
+of a zippered iteration, it will generate a conceptually infinite
+number of iterations (though practically, the behavior is undefined
+once it approaches the extreme values that its index type can store).
```

Comment:
I would prefer dropping "though practically" and the parens. In other words, making "undefined" a part of the official semantics. Alternatively place it into a formerly-"cray implementation" section.

Though practically, feel free to merge as-is and leave this change for another day.

Reply:
Good idea.